src/app/EventManagement.cpp

```diff
@@ -661,7 +678,7 @@ CHIP_ERROR EventManagement::CopyEventsSince(const TLVReader & aReader, size_t aD
 }
 
 CHIP_ERROR EventManagement::FetchEventsSince(TLVWriter & aWriter, ClusterInfo * apClusterInfolist, EventNumber & aEventMin,
-                                             size_t & aEventCount)
+                                             size_t & aEventCount, const FabricIndex & aFabricIndex)
```

Comment:
No, my question was why the last arg here is not `FabricIndex aFabricIndex`.  As in, why are we passing a reference, not a value.

Reply:
I think it is good to pass reference, update it. thanks